ingestr/src/sources.py

```diff
@@ -37,6 +38,7 @@
 from ingestr.src.slack import slack_source
 from ingestr.src.stripe_analytics import stripe_source
 from ingestr.src.table_definition import table_string_to_dataclass
+from ingestr.src.tiktok_ads._init_ import tiktok_source
```

Comment:
If you change `ingestr/src/tiktok_ads/_init_.py` to `ingestr/src/tiktok_ads/__init__.py`, you will be able to treat `tiktok_ads` as a python module.

That way, the import statement can change to:
```python
from ingestr.src.tiktok_ads import tiktok_source
```